orders-core/src/main/java/io/helidon/examples/sockshop/orders/DefaultOrderRepository.java

```diff
@@ -18,9 +21,11 @@
  * API testing and quick demos.
  */
 @ApplicationScoped
+@Alternative
+@Priority(Interceptor.Priority.APPLICATION+1)
```

Comment:
Make this `APPLICATION - 10` and remove the `@Alternative` as it is not necessary for the default implementation. We just need to make it lower priority than the other impls.